build.gradle.kts

```diff
@@ -6,6 +6,7 @@ group = "com.mongodb"
 version = "0.0.1"
 
 plugins {
+    alias(libs.plugins.intellij) apply false
```

Comment:
This is because `gradlew tasks` was failing on main and a workaround was recommended [here](https://youtrack.jetbrains.com/issue/IDEA-339602/Gradle-IDEA-Ext-support-classloader-isolation).